runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js

```diff
@@ -567,13 +567,17 @@ class WorkItemDetailPage {
   clickStartCoding  (){
     return this.startCodingElement.click();
   }
-  startCodingDiv (){
-    return element(by.id("start-coding-link"));
+  startCodingDiv  (){
+    return element(by.css(".form-control-static.start-coding"));
```

Comment:
ID for Div was moved to be button ID. So the div has to be referred to by CSS, until they have an ID of their own. Kindly refer to the HTML changed in this regard.